src/ocean/util/app/DaemonApp.d

```diff
@@ -561,7 +565,18 @@ public abstract class DaemonApp : Application,
 
     override public void onSignal ( int signum )
     {
-        // Dummy implementation of the interface
+        switch ( signum )
+        {
+            case core.sys.posix.signal.SIGTERM:
```

Comment:
How will this change affect apps which already have their own handling for SIGTERM?

Reply:
It should not change, the subclass override should prevail.

Reply:
Is that desired, though? The special scheduler shutdown behaviour would not occur then.

Reply:
Can't really do anything for that case without risking breakage. Maybe I should just mention that in release notes, advising to do the same.

Reply:
This is kind of a special case, in that the method is already implemented (i.e. all user code _must_ override it) but has a dummy implementation, so almost all user code will not be calling `super.onSignal(signum)`.

I'm wondering whether this is a breaking change.

Reply:
I would assume that whoever is handling the `SIGTERM` already does `theScheduler.shutdown()` or `epoll.shutdown()` anyway (in which case it will be replaced by `theScheduler.shutdown()` when migrating to schedule. If they are handling SIGTERM without shutting down scheduler, they are not exiting cleanly anyway, and I don't see anything is broken.

Reply:
In another words: user code is not _required_ to call `super.onSignal`.

Reply:
I see. I think this should definitely be mentioned in the release notes and probably in the method doc as well.